thorlcr/activities/thdiskbaseslave.cpp

```diff
@@ -91,7 +92,7 @@ void CDiskPartHandlerBase::setPart(IPartDescriptor *_partDesc, unsigned partNoSe
         {
             unsigned subfile;
             unsigned lnum;
-            if (superFDesc->mapSubPart(partNoSerialized, subfile, lnum))
+            if (superFDesc->mapSubPart(which, subfile, lnum))
```

Comment:
This is the only significant change (and moving the which = to line 86).
The rest of the changes in this PR are removing the 2nd setPart parameter, which is now redundant.